packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -19,6 +24,11 @@ class LoggerRabbitMessage(RabbitMessageBase):
     messages: List[str]
 
 
+class EventRabbitMessage(RabbitMessageBase):
+    action: str
+    payload: Optional[Dict[str, Any]]
```

Comment:
what payload? Can you use some ``TypeDict`` and use ``Union`` instead of open Dict?

Reply:
Based on the action, a different set of payload keys will be provided. that's why it is not fixed here.